src/stratifier.c

```diff
@@ -625,7 +628,7 @@ static void generate_coinbase(ckpool_t *ckp, workbase_t *wb)
 	/* Coinb2 address goes here, takes up 23~25 bytes + 1 byte for length */
 
 	wb->coinb3len = 0;
-	wb->coinb3bin = ckzalloc(256 + wb->insert_witness * (8 + witnessdata_size + 2));
+	wb->coinb3bin = ckzalloc(512 + wb->insert_witness * (8 + witnessdata_size + 2));
```

Comment:
512 instead of 256 since adding another output?

Reply:
yes it gives some room at the expense of 256 bytes of memory